### PR TITLE
Remove repeated code by extracting a common method.

### PR DESCRIPTION
**Pull request by thehereward**

### CODE REVIEW

**Comment by hypeJunction** [comment]
Thank you for the pull requests. We have a somewhat strict policy regarding commit message format: http://learn.elgg.org/en/2.0/contribute/code.html#commit-message-format

This should be something like `chore(logrotate): remove repeated code by extracting a common method`

File: mod/logrotate/start.php [hunk 1]
```diff
@@ -64,28 +51,37 @@ function logrotate_archive_cron($hook, $entity_type, $returnvalue, $params) {
...
+
+	return $returnvalue . $resulttext;
+}
+
+
+/**
+ * @param $period
+ * @return int
+ */
+function getSecondsInPeriod($period)
```
Comment (mrclay) [inline mod/logrotate/start.php]: This should be named: `logrotate_get_seconds_in_period`.

File: mod/logrotate/start.php [hunk 2]
```diff
@@ -64,28 +51,37 @@ function logrotate_archive_cron($hook, $entity_type, $returnvalue, $params) {
...
+{
+	$secondsInDay = 86400;
```
Comment (mrclay) [inline mod/logrotate/start.php]: `$seconds_in_day`. We only use camelCase for class methods, snake_case for everything else.

**Comment by thehereward** [comment]
I've updated this to address your comments.

File: mod/logrotate/start.php [hunk 3]
```diff
@@ -64,28 +50,37 @@ function logrotate_archive_cron($hook, $entity_type, $returnvalue, $params) {
...
+
+	return $returnvalue . $resulttext;
+}
+
+
+/**
+ * @param $period
+ * @return int
+ */
+function logroate_get_seconds_in_period($period)
```
Comment (mrclay) [inline mod/logrotate/start.php]: logrotate misspelled :)

**Comment by thehereward** [comment]
So it was.
That's what comes of coding last thing on a Sunday night I guess... :-)

**Pull request merged**